src/GeneratorV2/Process/Overloaders.cs

```diff
@@ -169,6 +169,229 @@ public void WritePrologue(IndentedTextWriter writer, NameTable nameTable)
         }
     }
 
+    public sealed class VectorOverloader : IOverloader
+    {
+        private static readonly Regex VectorNameMatch = new Regex("([1-4])([f|d|h|i])v$", RegexOptions.Compiled);
```

Comment:
The `[f|d|h|i]` should be changed to `[fdhi]` as the `|` in there only means it can also match that caracter.